### PR TITLE
Api Doc Material

### PR DESCRIPTION
**Pull request by vhhieu95**
Api Doc Material

### CODE REVIEW

File: docs/api-references/material.md [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+        "id": 123,
+        "name": "Material",
+        "category_id": 2,
+        "price": 1000,
+        "description": "",
+        "image": "http://link-to-image.com/image_name.npg",
+        "category": {
+            "id": 9,
+            "name": "Balistreri, Schaden and Swaniawski"
+        }"supplier": {
```
Comment (ghost) [inline docs/api-references/material.md]: Incorrect JSON format.

**Review by ghost — COMMENTED** [review]
Please check my comment.

**Comment by vhhieu95** [comment]
I updated please check again. Thank You @at-vietpham

**Comment by ghost** [comment]
LGTM

**Pull request merged**